Add position attribute on button

GhBogdan97 opened the pull request:
Fixes Issue #7481 

Edit CSS for "Copy" button to be centered on Firefox

char0n commented:
@mathis-m, interested in lending second pair of eyes here and doing a community code review?

mathis-m commented:
@char0n @GhBogdan97 I do not see it fully resolved, if we touch this I think we should provide a more consistent behavior!
chrome:
![image](https://user-images.githubusercontent.com/11584315/136797563-73f96be0-6636-4e21-b030-528957644fcb.png)
firefox:
![image](https://user-images.githubusercontent.com/11584315/136797659-3001aee9-8ca1-4a10-9d38-9fae2580598d.png)

There are 2 things I would do, first center the svg path itself.
Second, use flex-box to center it within the button! I'll post a review soon, but the result looks like this:
![image](https://user-images.githubusercontent.com/11584315/136801970-fbc489ea-73c1-4f35-8295-a93426d59342.png)

mathis-m commented:
SVG:
before:
![image](https://user-images.githubusercontent.com/11584315/136802835-67406c52-989b-4f1e-851e-c2f6a4deb086.png)

after:
![image](https://user-images.githubusercontent.com/11584315/136802732-23a088a4-0aff-46ec-921a-a0c5b970c626.png)

And I removed the hacky paddings and just used flex-box to center that svg-button.

GhBogdan97 commented:
Thank you for the review. I've added the changes you've mentioned.

I've added the flex/align/justify attributes on the "Download" button as well, since it was slightly misaligned.
